src/modules/commentPopup.js

```diff
@@ -0,0 +1,86 @@
+import { addComment, getComments, countComments } from './comment.js';
+
+const commentPopup = async (popupComment, movie, num) => {
+  popupComment.innerHTML += `
+    <div class="popup">
+    <span class="close-btn">X</span>
+        <div class="popup-content">
+            
+            <img src="${movie.image.medium}"/>
+            <h1>${movie.name}</h1>
+            <div class="popup-detail">
+            <p>Genre : ${movie.genres[0]}</p>
+            <p> Show ended: ${movie.ended} </p>
+            <p>Language : ${movie.language}</p>
+            <p>Duration: ${movie.runtime}</p>
+            </div>
+            <h4 class='count-display'>Comments ${num} </h4>
+            <div class='comments-div' id="com-div">
+
+            </div>
+            <h4>Add a comment</h4>
+            <input type="text" placeholder="Your name" name="name" class="comment-input"/>
+            <input type="textarea" placeholder="Your insights" name="comment" class="comment-input"/>
```

Comment:
- [ ] At a point in time, I discovered that it was really difficult to see these two fields. They are hidden hence I couldn't add a comment. Let's make sure that users have a good experience when using our app.




![Screenshot from 2022-10-28 15-57-40](https://user-images.githubusercontent.com/63087491/198664688-93269ef1-50b9-41d2-a6fb-456f4ba3e47e.png)